app/cdap/components/GridTable/index.tsx

```diff
@@ -41,9 +41,7 @@ import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router';
 import { flatMap } from 'rxjs/operators';
 import { objectQuery } from 'services/helpers';
-
 import styled from 'styled-components';
```

Comment:
as we don't have any feature related code here we can remove this formatting changes as well so that review files count will reduce